src/Area/Area.ts

```diff
@@ -0,0 +1,7 @@
+export class Area {
+    Id: string;
+    Name: string;
+    Version: number;
```

Comment:
Date or number?

Reply:
Typescript's Dates seem to be just ticks under the hood. I couldn't find a good way to do something like GetUTCNow as a date. Please let me know if I'm missing something, because I'd much rather that be a date.